### ISSUE TITLE
Column vector defaults to list view rather than table view

### ISSUE THREAD

**Issue by scspaeth** [jmoenig/Snap--Build-Your-Own-Blocks#1160]
The new Table view automatically renders a row vector as a 1Xn table. The one dimensional column vector, however, automatically renders as a list view. They render correctly in table view if you specify Table view or 'open in dialog.' 
For consistency, shouldn't the column vector default to a mX1 table view?
![screen shot 2016-03-06 at 1 19 19 pm](https://cloud.githubusercontent.com/assets/2250050/13556026/1e7440d2-e39e-11e5-9f53-c51d0b59db03.png)

**Comment by jmoenig** [comment]
Yep, that's actually intentional, and I swear there was a compelling reason for this, but I've forgotten it right now. Will investigate, thanks!
